addon/background.js

```diff
@@ -21,20 +21,20 @@ browser.contextMenus.create({
   documentUrlPatterns: ["<all_urls>"]
 });
 
-browser.contextMenus.onClicked.addListener((info, tab) => {
+browser.contextMenus.onClicked.addListener(async (info, tab) => {
   let url = info.linkUrl || tab.url;
   // let title = info.linkText || tab.title || "Page";
-  browser.sidebarAction.open().then(() => {
-    return browser.windows.getCurrent();
-  }).then((windowInfo) => {
+  try {
+    await browser.sidebarAction.open();
+    const windowInfo = await browser.windows.getCurrent();
     let desktop = !!desktopHostnames[(new URL(url)).hostname];
     let message = {type: "browse", url, windowId: windowInfo.id, desktop};
     return retry(() => {
       return browser.runtime.sendMessage(message);
     }, {times: 3, wait: 50});
-  }).catch((error) => {
+  } catch (error) {
     console.error("Error setting panel to page:", error);
```

Comment:
We might want to consider if these errors are reported fine on their own. It's not great with `.catch()`, but maybe this works fine? Or not... worth experimenting I guess.